nn/conformer.py

```diff
@@ -4,12 +4,11 @@
 """
 
 from typing import Tuple, List, Union
+from .. import nn
+from . import LayerRef
```

Comment:
Do you think it is a good idea to have `LayerRef` as an explicit import? Why not just use `nn.LayerRef`?

Reply:
I just thought that I can save some space when defining the parameters. But I can just use `nn.LayerRef`

Reply:
Yes use `nn.LayerRef`. This will also save you this line of code here.